cmd/cogito/main.go

```diff
@@ -50,21 +50,14 @@ func mainErr(in io.Reader, out io.Writer, logOut io.Writer, args []string) error
 	})
 	log.Info(cogito.BuildInfo())
 
-	ghAPI := os.Getenv("COGITO_GITHUB_API")
-	if ghAPI != "" {
-		log.Info("endpoint override", "COGITO_GITHUB_API", ghAPI)
-	} else {
-		ghAPI = github.API
-	}
-
```

Comment:
I removed the environment-based API endpoint override. There was only one usage of it (in`main_test.go`). I replaced that usage with the custom test URL in the `github_api_endpoint` source config. The test still works.

Let me know if you have other plans, and I will revert this change.